x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/maintenance_window_callout/use_fetch_active_maintenance_windows.ts

```diff
@@ -11,13 +11,14 @@ import { useAppToasts } from '../../../../common/hooks/use_app_toasts';
 import * as i18n from './translations';
 import { fetchActiveMaintenanceWindows } from './api';
 
-export const useFetchActiveMaintenanceWindows = () => {
+export const useFetchActiveMaintenanceWindows = (enabled: boolean) => {
```

Comment:
nit: Explicitly using supported parameters `UseQueryOptions` from `UseQueryOptions` and picking only `enabled` field helps to improve readability and extend it later if necessary wit minimal changes.

```suggestion
export const useFetchActiveMaintenanceWindows = ({ enabled }: Pick<UseQueryOptions, 'enabled'>) => {
```